configs/taco/GU.yaml

```diff
@@ -6,7 +6,10 @@ links:
 
 - name: secondary
   url: https://drive.google.com/file/d/1FVW6sNbdgyV5-CYkOorYvOGivkmOED8t/view
-
+  file: pdf
```

Comment:
we also need to change the URLs here: this one should be `https://drive.google.com/uc?export=download&id=1FVW6sNbdgyV5-CYkOorYvOGivkmOED8t` and the tertiary one should be `https://drive.google.com/uc?export=download&id=1n-vrUGzEzs7rjcxXBFEbIMmeUQykuyO9`